scripts/release.bat

```diff
@@ -32,11 +32,10 @@ set VERSION=%2
 IF "%VERSION%"=="2015" (
     set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvc*.dll"
 ) ELSE (
-
-    IF EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\" (
-        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\msvc*.dll"
+    IF EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\" (
+        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\msvc*.dll"
 		) ELSE (
-        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.13.26020\x86\Microsoft.VC141.CRT\msvc*.dll"
+        set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\14.14.26405\x86\Microsoft.VC141.CRT\msvc*.dll"
```

Comment:
Does this mean that Appveyor upgraded their 2017 version?

Reply:
@axic Yes, they updated to VS Community 2017 15.7.3 (https://www.appveyor.com/docs/build-environment/#visual-studio-2017). I can check if there's a way to fix the version.

Reply:
Would `set "DLLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Redist\MSVC\*\x86\Microsoft.VC141.CRT\msvc*.dll"` work?

Reply:
@chriseth I'll give it a try. AFAIK, there's no way to specify a fixed VS version in the AppVeyor build environment.